examples/gltf/app.js

```diff
@@ -42,30 +43,28 @@ const DEFAULT_OPTIONS = {
 };
 
 async function loadGLTF(urlOrPromise, gl, options = DEFAULT_OPTIONS) {
-  const promise = urlOrPromise instanceof Promise
-   ? urlOrPromise
-   : window.fetch(urlOrPromise).then(res => urlOrPromise.endsWith('.gltf') ? res.json() : res.arrayBuffer());
-  let gltfParser;
+  const promise =
+    urlOrPromise instanceof Promise
+      ? urlOrPromise
+      : window
+          .fetch(urlOrPromise)
```

Comment:
- The suffix switching should no longer be needed. 
- `GLTFParser.parse()` now accepts both binary and text and checks whether the file is GLB or JSON, and does any necessary text and JSON decoding as necessary.

Reply:
ok thanks.